test-engineering/contract-tests/README.md

```diff
@@ -0,0 +1,59 @@
+# contile-contract-tests
+
+This directory contains the automated contract test suite for the Mozilla Tile 
+Service (MTS). Passing contract tests are a prerequisite for moving to the next 
+phase in the rollout plan. The contract test framework was originally developed 
+in isolation, see [contile-integration-tests][contract-test-repo].
+
+## Overview
+
+The contract test suite is designed to be set up as a `docker-compose` CI
+workflow. The following sections as well as the sequence diagram below describe 
+the individual components of the suite.
+
+![Sequence diagram of the integration tests][sequence_diagram]
+
+To run the contract tests locally, execute the following from the repository root:
+
+```text
+docker compose \ 
+-f test-engineering\contract-tests\docker-compose.yml \ 
+up --abort-on-container-exit --build
+```
+
+### partner
+
+The `partner` directory contains a Python-based web service. The HTTP API of
+this service implements the API specification of the partner API that MTS
+connects to when requesting tiles to pass along to Firefox for display.
+
+When a client sends a request to the MTS, information about the client's form
+factor and OS family are parsed from the `User-Agent` header. Then, when the MTS
+sends a request to the partner API the form factor and OS family information is
+included in the query parameters. We leverage this behavior to map requests from
+a client to specific responses from the partner API. We can control not only the
+response content, but also the response status code, response headers and even
+delay the response for a period of time, which allows us to effectively test the
+MTS.
+
+### client
+
+The `client` directory contains a Python-based test framework for the
+contract tests. The HTTP client used in the framework requests tiles from the
+MTS and performs checks against the responses. The framework implements response
+models for the MTS API.
+
+### volumes
+
+The `volumes` directory contains subdirectories which will be mounted as
```

Comment:
We've added a `volumes` directory at the contile-integration-tests repo root with example files for documentation purposes: https://github.com/mozilla-services/contile-integration-tests/tree/main/volumes

From what I can tell this pull-request moves the actual `volumes` for the tests here:
https://github.com/mozilla-services/contile/tree/main/contract-tests/volumes

I think we can delete the contile-integration-tests `volumes` directory and its contents and only keep actual volumes used in the contract-tests on CI. Is that what your change is doing?

Reply:
Yes.
More specifically, because the contile-integration-test volumes files had the same names as the contract-test files, there were merge conflicts when I merged repositories. I resolved the conflicts by picking the contile files.

Reply:
Great, thank you! 👍🏻